### DISCUSSION TITLE
What does this mean

### DISCUSSION THREAD

**Discussion by Xlostsoul666** [termux/termux-app#3716] [General]
E: Failed to fetch https://deb.kcubeterm.me/termux-main/dists/stable/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
E: The repository 'https://deb.kcubeterm.me/termux-main stable InRelease' is not signed.

**Comment by twaik** [comment]
**Termux on Google Play Store is OUTDATED** 🪦
Termux and its plugins are no longer updated on the Google Play Store. This is due to API requirements that introduce new restrictions, which break Termux's basic functionality.
Please switch to the F-Droid version!
Direct link: <https://f-droid.org/en/packages/com.termux>
**Before** you install it, You **must** __fully uninstall__ current Termux and all plugins. If you don't, you will get a cryptic `app not installed` error!
(that means uninstalling the main app, as well as Termux:API, :Styling, :Widget, :Task, :Float, :Boot)
If you need help with backing up your files:
<https://wiki.termux.com/wiki/Backing_up_Termux>

**Reply by twaik** [reply]
After installing Termux and opening it, you should do the following:

**1.** `termux-change-repo` (to select a nearby mirror to get updates from):
- `SPACE` toggles highlighted options;
- `ENTER` moves to the next screen;
Choose the first/topmost option on each screen. _(If that one fails, try another.)_
**2.** `pkg update && pkg upgrade` (to update your package database, then upgrade all packages)
On each prompt, press `y` and `Enter` to install newest versions available
**3.** `exit` (to exit Termux, so you can restart it yourself, all updated!)

**Reply by twaik** [reply]
The discussion can be closed...
